app/functions/functions_db.php

```diff
@@ -0,0 +1,40 @@
+<?php
+require_once('config/config.php');
+
+$mysqli = mysqli_connect($db['host'], $db['user'], $db['password'], $db['database']);
+
+mysqli_set_charset($mysqli, 'utf8');
+
+if (!$mysqli) {
+    echo('Ошибка подключения к базе данных' . mysqli_connect_error());
+} else {
+    $connection = $mysqli;
+}
```

Comment:
Давай постараемся избегать if **else** конструкций.

Если нет коннекта к БД, предлагаю просто упасть с ошибкой с помощью die().

```suggestion
if (false === $mysqli) {
    die('Ошибка подключения к базе данных: ' . mysqli_connect_error());
}
// дальше выполнение
```